test.js

```diff
@@ -175,19 +179,14 @@ test('rename filenames using a function', async t => {
 	t.is(read(t.context.tmp, 'source/bar.js'), read(t.context.tmp, 'destination/subdir/source/prefix-bar.js'));
 });
 
-test('cp-file errors are not glob errors', async t => {
-	const error = await t.throwsAsync(cpy('license', t.context.EPERM), /EPERM/);
-	t.notRegex(error.message, /glob/);
-});
-
 test('cp-file errors are CpyErrors', async t => {
-	const error = await t.throwsAsync(cpy('license', t.context.EPERM), /EPERM/);
-	t.true(error instanceof CpyError);
+	const cpy = cpyMockedError('cp-file');
+	await t.throwsAsync(cpy('license', t.context.dir), {message: /cp-file/, instanceOf: CpyError});
 });
 
```

Comment:
These tests are checking for proper handling of exceptions thrown by dependencies.  Previously, the test were implemented with the contrived scenario of a no-access directory which would force these modules to fail: `fs.mkdirSync(t.context.EPERM, 0);` on *nix systems.  This has been replaced with mocking of the dependencies to fail immediately.  With this, the additional `cp-file + glob` test also becomes redundant. 

While a similarly contrived example could have been created for Windows ( e.g. writing to a known folder ) successful test execution would have been dependent on the environment the test was run in ( e.g. elevated shell ).